tests/unit/src/Formatter/Http/JsonTest.php

```diff
@@ -30,7 +31,19 @@ public function testItReturnsAValidJsonResponse(): void
 
         $this->assertSame(HealthCheck::STATUS_UNHEALTHY, $body->status);
         $this->assertNotEmpty($body->dependencies);
-        $this->assertSame($depenencyName, $body->dependencies[0]->name);
+        $this->assertSame($dependencyName, $body->dependencies[0]->name);
         $this->assertFalse($body->dependencies[0]->healthy);
     }
+
+    public function testItReturnsAnErrorStatusCode(): void
```

Comment:
Can we name this `testItReturnsInternalServerErrorStatusCode`? As current name sounds like it could be any status code.